### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by AlwaysHungrie**

### CODE REVIEW

File: src/components/SearchResultsCard/SearchResultCard.jsx [hunk 1]
```diff
@@ -59,10 +60,99 @@ const useStyles = makeStyles(theme => ({
...
+        navigator.clipboard.writeText(phone)
+        setDialogMessage("Phone Number Copied to Clipboard")
+        setDialogOpen(true)   
+    }
+
+    const copyLink = () => {
+        // @TODO generate link to this page
+        navigator.clipboard.writeText("https://covid-resources-india-cef91.web.app/")
+        setDialogMessage("Link Copied to Clipboard")
+        setDialogOpen(true)
```
Comment (varora1406) [inline src/components/SearchResultsCard/SearchResultCard.jsx:153]: I am thinking, to make it complete it write now.
Can we copy details of card?
whatever we show in UI right now

what say?

File: src/containers/SearchPage/index.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ function SearchPage() {
   const { data, called } = useQuery(GET_SEARCH(state?.searchInputs?.state, state?.searchInputs?.city, state?.searchInputs?.requirement))
   if (state?.searchInputs) {
     currentData = data?.workspace?.tickets?.edges || [];
+    console.log(currentData)
```
Comment (varora1406) [inline src/containers/SearchPage/index.tsx:17]: not needed now, right?

**Review by varora1406 — APPROVED** [review]

**Review by varora1406 — CHANGES_REQUESTED** [review]
please look at changes suggested

File: src/components/SearchResultsCard/SearchResultCard.jsx (hunk 1 above)
Comment (AlwaysHungrie) [reply src/components/SearchResultsCard/SearchResultCard.jsx:153]: yeah can do that
would be more helpful

File: src/containers/SearchPage/index.tsx (hunk 2 above)
Comment (AlwaysHungrie) [reply src/containers/SearchPage/index.tsx:17]: yeah, sorry

**Pull request merged**